integration/system/stats_test.go

```diff
@@ -202,6 +193,37 @@ func getNetworkStats(c *testing.T, id string) map[string]types.NetworkStats {
 	return st.Networks
 }
 
+func RuntimeIsWindowsContainerd() bool {
+	return os.Getenv("DOCKER_WINDOWS_CONTAINERD_RUNTIME") == "1"
+}
+
+func TestAPIStatsNetworkStatsVersioning(t *testing.T) {
+	// Windows doesn't support API versions less than 1.25, so no point testing 1.17 .. 1.21
+	skip.If(t, !testEnv.IsLocalDaemon())
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
+
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	cID := container.Run(ctx, t, apiClient, container.WithImage("busybox"), container.WithCmd("sleep", "20"))
+	wg := sync.WaitGroup{}
+
+	for i := 17; i <= 21; i++ {
```

Comment:
- Oh! Nice one; I would've overlooked that this was old API versions here. Perhaps we should add a TODO to this function to remove these parts once we fully drop API < 1.24 (related to https://github.com/moby/moby/pull/46887)